pastis/launchers/run_luvoir_efields.py

```diff
@@ -12,15 +17,51 @@
     fpath = CONFIG_PASTIS.get('LUVOIR', 'harris_data_path')  # path to Harris spreadsheet
     pad_orientations = np.pi / 2 * np.ones(120)
 
-    DM_SPEC = (fpath, pad_orientations, False, True, False)
+    DM_SPEC = (fpath, pad_orientations, True, False, False)
     # DM_SPEC = tuple or int, specification for the used DM -
     #    for seg_mirror: int, number of local Zernike modes on each segment
-    #    for harris_seg_mirror: tuple (string, array, bool, bool, bool), absolute path to Harris spreadsheet, pad orientations, choice of Harris mode sets (thermal, mechanical, other)
+    #    for harris_seg_mirror: tuple (string, array, bool, bool, bool),
+    #    absolute path to Harris spreadsheet, pad orientations, choice of Harris mode sets (thermal, mechanical, other)
     #    for zernike_mirror: int, number of global Zernikes
 
     # First generate a couple of matrices
     run_matrix = MatrixEfieldLuvoirA(which_dm=DM, dm_spec=DM_SPEC, design=APLC_DESIGN,
-                                     initial_path=CONFIG_PASTIS.get('local', 'local_data_path'))
+                                     calc_science=True, calc_wfs=True,
+                                     initial_path=CONFIG_PASTIS.get('local', 'local_data_path'),
+                                     norm_one_photon=True)
     run_matrix.calc()
     dir_run = run_matrix.overall_dir
     print(f'All saved to {dir_run}.')
+
+
+    # Instantiate the LUVOIR telescope
+    optics_input = os.path.join(util.find_repo_location(), CONFIG_PASTIS.get('LUVOIR', 'optics_path_in_repo'))
+    coronagraph_design = CONFIG_PASTIS.get('LUVOIR', 'coronagraph_design')
+    sampling = CONFIG_PASTIS.getfloat('LUVOIR', 'sampling')
+    tel = LuvoirA_APLC(optics_input, coronagraph_design, sampling)
```

Comment:
This is duplicate code, as you can assess the simulator that has been used in `MatrixEfieldLuvoirA` a couple of lines above with `MatrixEfieldLuvoirA.simulator`.